lib/carin_for_blue_button_test_kit/client/c4bb_client_test_suite.rb

```diff
@@ -0,0 +1,124 @@
+require 'inferno/dsl/oauth_credentials'
+require_relative 'ext/inferno_core/record_response_route'
+require_relative 'ext/inferno_core/runnable'
+require_relative 'ext/inferno_core/request'
+require_relative 'urls'
+require_relative 'mock_server'
+require_relative 'tags'
+require_relative 'collection'
+require_relative 'client_validation_test'
+
+require_relative 'claim_data_request_tests/initial_scratch_storing'
+require_relative 'claim_data_request_tests/initial_wait_test'
+require_relative 'claim_data_request_tests/patient_claims_data_request_test'
+require_relative 'claim_data_request_tests/coverage_claims_data_request_test'
+require_relative 'claim_data_request_tests/organization_claims_data_request_test'
+require_relative 'claim_data_request_tests/practitioner_claims_data_request_test'
+require_relative 'claim_data_request_tests/relatedperson_claims_data_request_test'
+require_relative 'claim_data_request_tests/eob_inpatient_claims_data_request_test'
+require_relative 'claim_data_request_tests/eob_outpatient_claims_data_request_test'
+require_relative 'claim_data_request_tests/eob_oral_claims_data_request_test'
+require_relative 'claim_data_request_tests/eob_pharmacy_claims_data_request_test'
+require_relative 'claim_data_request_tests/eob_professional_claims_data_request_test'
+
+require_relative 'claim_data_request_tests/client_claims_data_attestation_test'
+
+module CarinForBlueButtonTestKit
+  class CarinForBlueButtonClientSuite < Inferno::TestSuite
+    extend MockServer
+    extend ClientValidationTest
+
+    id :c4bb_client
```

Comment:
let's include the IG version in the id and in other places like the suite class and title so that its clear when comparing to server suites which one this corresponds to.